backend/schema/User.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  {
+    name: {
```

Comment:
name or id?

Reply:
ID is autogenerated as `_id`. Name is the user's actual name.

Reply:
i mean like spotify id or email or something